### PR TITLE
fix persistent login

### PR DESCRIPTION
**Pull request by Daniel-Majunke**

### CODE REVIEW

**Comment by Daniel-Majunke** [comment]
#680

File: src/TrueOrFalse.Frontend.Web.Nuxt/pages/user/[name]/[id].vue [hunk 1]
```diff
@@ -86,7 +86,7 @@ onBeforeMount(() => {
 
 const { data: wuwi, refresh: refreshWuwi } = await useLazyFetch<Wuwi>(`/apiVue/User/GetWuwi/${route.params.id ? route.params.id : userStore.id}`, {
     credentials: 'include',
-    mode: 'no-cors',
+    mode: 'cors',
```
Comment (Daniel-Majunke) [inline src/TrueOrFalse.Frontend.Web.Nuxt/pages/user/[name]/[id].vue:89]: revert

File: src/TrueOrFalse.Frontend.Web.Nuxt/pages/user/[name]/[id].vue [hunk 2]
```diff
@@ -65,7 +65,7 @@ interface ProfileData {
 
 const { data: profile, refresh: refreshProfile } = await useFetch<ProfileData>(`/apiVue/User/Get/${route.params.id ? route.params.id : userStore.id}`, {
     credentials: 'include',
-    mode: 'no-cors',
+    mode: 'cors',
```
Comment (Daniel-Majunke) [inline src/TrueOrFalse.Frontend.Web.Nuxt/pages/user/[name]/[id].vue:68]: revert

File: src/TrueOrFalse.Frontend.Web/Program.cs [hunk 3]
```diff
@@ -80,9 +79,15 @@
         options.DefaultSignInScheme = CookieAuthenticationDefaults.AuthenticationScheme;
         options.DefaultChallengeScheme = CookieAuthenticationDefaults.AuthenticationScheme;
     })
-    .AddCookie();
+    .AddCookie(options =>
```
Comment (Daniel-Majunke) [inline src/TrueOrFalse.Frontend.Web/Program.cs:82]: remove authentication

File: src/TrueOrFalse.Frontend.Web/Program.cs [hunk 4]
```diff
@@ -98,12 +103,12 @@
 if (!env.IsDevelopment())
 {
     app.UseExceptionHandler("/Home/Error");
+    app.UseHsts();
```
Comment (Daniel-Majunke) [inline src/TrueOrFalse.Frontend.Web/Program.cs:106]: remove

File: src/TrueOrFalse.Frontend.Web/Program.cs [hunk 5]
```diff
@@ -129,25 +134,27 @@
 
 app.UseSession();
 app.UseRouting();
+
```
Comment (Daniel-Majunke) [inline src/TrueOrFalse.Frontend.Web/Program.cs:137]: remove

File: src/TrueOrFalse.Frontend.Web/Views/ApiVueApp/AppController.cs [hunk 6]
```diff
@@ -35,37 +49,37 @@ public record struct ActivityPoints(
     [HttpGet]
     public GetCurrentUserResult GetCurrentUser()
     {
-        var sessionUser = _vueSessionUser.GetCurrentUserData();
+        var currentUser = _vueSessionUser.GetCurrentUserData();
```
Comment (Daniel-Majunke) [inline src/TrueOrFalse.Frontend.Web/Views/ApiVueApp/AppController.cs:52]: should rename vueSessionUser to FrontendUserData.Get() something like that

File: src/TrueOrFalse.Frontend.Web/Views/ApiVueApp/Middleware/AuthController.cs [hunk 7]
```diff
@@ -4,8 +4,6 @@ namespace VueApp;
 
 public class MiddlewareAuthController(SessionUser _sessionUser) : Controller
 {
```
Comment (Daniel-Majunke) [inline src/TrueOrFalse.Frontend.Web/Views/ApiVueApp/Middleware/AuthController.cs:6]: revert code

File: src/TrueOrFalse/Infrastructure/Login/Model/Ops/LoginFromCookie.cs [hunk 8]
```diff
@@ -2,32 +2,28 @@
...
-            return false;
+            return;
 
         var user = userReadingRepo.GetById(cookieValues.UserId);
         if (user == null)
-            return false;
+            return;
 
+        sessionUser.Login(user);
         persistentLoginRepo.Delete(persistentLogin);
```
Comment (Daniel-Majunke) [inline src/TrueOrFalse/Infrastructure/Login/Model/Ops/LoginFromCookie.cs:25]: check min db if it rly gets deleted

File: src/TrueOrFalse.Frontend.Web/Program.cs (hunk 3 above)
Comment (tj-mai) [reply src/TrueOrFalse.Frontend.Web/Program.cs:82]: can't be deleted, needs for logout

File: src/TrueOrFalse.Frontend.Web/Views/ApiVueApp/Middleware/AuthController.cs
Comment (Daniel-Majunke) [inline src/TrueOrFalse.Frontend.Web/Views/ApiVueApp/Middleware/AuthController.cs:1]: Please switch the two attributes to [AccessOnlyAsLoggedIn] [AccessOnlyAsAdmin]

**Review by Daniel-Majunke — COMMENTED** [review]
lgm

**Pull request merged**